js/modules/markjoseph.js

```diff
@@ -0,0 +1,111 @@
+export default {
+    state: {
+        profile: {
+            username: "your-github-username",
+            fullName: "Your Full Name",
```

Comment:
Put your full name here.

```suggestion
            fullName: "Markjoseph Ballecer",
```